<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForgotPasswordWindow</class>
 <widget class="QDialog" name="ForgotPasswordWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="EmailEnteryLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 36px;</string>
   </property>
   <property name="text">
    <string>Enter your email:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EmailLine">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 24px;
border-radius: 10px;
background-color: rgb(231, 231, 231);
border: 1px solid rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px;
border-radius: 10px;
background-color: rgb(231, 231, 231);
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 30px;</string>
   </property>
   <property name="text">
    <string>Send password again in {time}.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
